### PR TITLE
docs: add sync status feature guide

### PR DESCRIPTION
**Pull request by hsheth2**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by shirshanka** [comment]
looks like doc build failed

File: docs/sync-status.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+The sync status feature is enabled by default and does not require any special setup.
+
+## Using Sync Status
+
+The DataHub UI will display the sync status in the top right corner of the page.
+
+![sync status](https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png)
+
+The last synchronized date is computed as the most recent edit to the entity, excluding edits done through the UI.
+We'll automatically assign a color based on the sync status recency, with green indicating a fresh entity and red indicating a stale one.
```
Comment (maggiehays) [inline docs/sync-status.md:22]: Let's elaborate here on the underlying business logic that determines the colors.

File: docs/sync-status.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The sync status feature is enabled by default and does not require any special setup.
+
+## Using Sync Status
+
+The DataHub UI will display the sync status in the top right corner of the page.
+
+![sync status](https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png)
+
+The last synchronized date is computed as the most recent edit to the entity, excluding edits done through the UI.
```
Comment (maggiehays) [inline docs/sync-status.md:21]: Let's elaborate on this a little bit - what counts as an edit? What happens in there's an ingestion run, but no net-new edits? if someone sees that something is out of sync, what actions can they take to get it back into sync?

File: docs/sync-status.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Sync Status Setup, Prerequisites, and Permissions
+
+The sync status feature is enabled by default and does not require any special setup.
+
+## Using Sync Status
+
+The DataHub UI will display the sync status in the top right corner of the page.
+
+![sync status](https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png)
```
Comment (maggiehays) [inline docs/sync-status.md:19]: Small nit-pick for consistency with other guides & improved readability - please use HTML to embed images so we can size down (70%) & center-align: 

```
<p align="center">
  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status.png"/>
</p>
```

File: docs/sync-status.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+The last synchronized date is computed as the most recent edit to the entity, excluding edits done through the UI.
+We'll automatically assign a color based on the sync status recency, with green indicating a fresh entity and red indicating a stale one.
+You can hover over the sync status message in the UI to view the exact timestamp of the most recent sync.
+
+![hover card](https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/sync-status-hover-card.png)
```
Comment (maggiehays) [inline docs/sync-status.md:25]: Same thing here - please use HTML for consistency in 70% sizing & center alignment

**Review by maggiehays — CHANGES_REQUESTED** [review]

**Comment by laulpogan** [comment]
@shirshanka per @hsheth2 this PR is good to go, we just need your approval

**Review by maggiehays — APPROVED** [review]

**Pull request merged**